ci-operator/step-registry/cucushift/helper-service/build-info/cucushift-helper-service-build-info-commands.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [[ -n "${RELEASE_IMAGE_LATEST}" ]]; then
+    echo "Release info for: ${RELEASE_IMAGE_LATEST}"
+    oc adm release info "${RELEASE_IMAGE_LATEST}" || true
+    PAYLOAD_VERSION="$(oc adm release info "${RELEASE_IMAGE_LATEST}" --output=jsonpath="{.metadata.version}")"
+    echo "${PAYLOAD_VERSION}" > "${ARTIFACT_DIR}/ocp-build-info"
```

Comment:
Looks good, is it necessary that we also add such info for upgrade jobs

Reply:
upgrade needs extra effort than e2e, will do that in another PR.

Reply:
Add the info for upgrade in https://github.com/openshift/release/pull/30881